src/node.cc

```diff
@@ -4469,6 +4469,12 @@ void Init(int* argc,
         SafeGetenv("NODE_PENDING_DEPRECATION", &text) && text[0] == '1';
   }
 
+  {
+    std::string text;
+    config_expose_http2 =
+      !(SafeGetenv("NODE_NO_HTTP2", &text) && text[0] == '1');
```

Comment:
I think we had a similar discussion like this, but this would currently make a distinction between `NODE_NO_HTTP2=19` and `NODE_NO_HTTP2=20`. Why not just `… && text == "1"`?